src/box/lua/config/instance_config.lua

```diff
@@ -1246,6 +1246,21 @@ return schema.new('instance_config', schema.record({
                                 type = 'string',
                             }),
                         }),
+                        lua_eval = schema.array({
+                            items = schema.scalar({
+                                type = 'string',
+                            }),
+                        }),
```

Comment:
Unlike `lua_call` and `sql`, which may be updated in a future with more granularity, `lua_eval` doesn't seem as something that may be granular. So, it means that `null` and `[all]` are the only valid values now and in a future. Shouldn't we make it boolean instead?